app/dynamic/repository/object_repository.py

```diff
@@ -68,6 +68,31 @@ def get_by_object_type_and_uuid(self, object_type: str, uuid: UUID) -> Optional[
         stmt = select(ObjectsTable).filter(ObjectsTable.UUID == uuid).filter(ObjectsTable.Object_Type == object_type)
         return self.fetch_first(stmt)
 
+    @staticmethod
+    def next_valid_by_uuid_query(object_uuid: UUID):
+        reference_obj = (select(ObjectsTable).filter(ObjectsTable.UUID == object_uuid)).subquery()
+
+        stmt = (
+            select(ObjectsTable)
+            .options(selectinload(ObjectsTable.ObjectStatics))
+            .join(ObjectsTable.ObjectStatics)
+            .filter(ObjectsTable.Object_ID == reference_obj.c.Object_ID)
```

Comment:
Dit is op `Code` en niet op `Object_ID`.

`Object_ID` is alleen uniek in combinatie met `Object_Type`